doc/sphinxman/source/freq.rst

```diff
@@ -50,7 +50,21 @@ the scenes, :py:func:`~psi4.frequency` is a light wrapper over
 :py:func:`~psi4.hessian` that computes the Hessian then adds a
 thermochemical analysis.
 
-.. autofunction:: psi4.frequency(name [, molecule, return_wfn, func, mode, dertype, irrep])
+Visualization of Normal Modes
+-----------------------------
 
-.. autofunction:: psi4.hessian(name [, molecule, return_wfn, func, dertype, irrep])
+|PSIfour| has the ability to export a Molden file that stores information about
```

Comment:
Did the following lines get deleted by accident? Not a show-stopper, I can add them back in later.

```
.. autofunction:: psi4.frequency(name [, molecule, return_wfn, func, mode, dertype, irrep])

.. autofunction:: psi4.hessian(name [, molecule, return_wfn, func, dertype, irrep])
```